cluster/src/test/java/com/hartwig/pipeline/tertiary/orange/OrangeTest.java

```diff
@@ -60,31 +60,31 @@ protected List<String> expectedInputs() {
+                input(TestInputs.SOMATIC_BUCKET + "/chord/tumor_chord_prediction.txt", "tumor_chord_prediction.txt"),
+                input(TestInputs.REFERENCE_BUCKET + "/bam_metrics/reference" + BAM_METRICS_SUMMARY_TSV, "reference" + BAM_METRICS_SUMMARY_TSV),
+                input(TestInputs.TUMOR_BUCKET + "/bam_metrics/tumor" + BAM_METRICS_SUMMARY_TSV, "tumor" + BAM_METRICS_SUMMARY_TSV),
+                input(TestInputs.REFERENCE_BUCKET + "/bam_metrics/reference" + BAM_METRICS_FLAG_COUNT_TSV, "reference" + BAM_METRICS_FLAG_COUNT_TSV),
+                input(TestInputs.TUMOR_BUCKET + "/bam_metrics/tumor" + BAM_METRICS_FLAG_COUNT_TSV, "tumor" + BAM_METRICS_FLAG_COUNT_TSV),
                 /*
                 input("run-reference-test/bam_metrics/results/reference.wgsmetrics", "reference.wgsmetrics"),
                 input("run-tumor-test/bam_metrics/results/tumor.wgsmetrics", "tumor.wgsmetrics"),
                 input("run-reference-test/flagstat/reference.flagstat", "reference.flagstat"),
                 input("run-tumor-test/flagstat/tumor.flagstat", "tumor.flagstat"),
                  */
-                input(expectedRuntimeBucketName() + "/sage_germline/results/tumorsage.gene.coverage.tsv", "tumorsage.gene.coverage.tsv"),
-                input(expectedRuntimeBucketName() + "/sage_somatic/results/referencesage.bqr.png", "referencesage.bqr.png"),
-                input(expectedRuntimeBucketName() + "/sage_somatic/results/tumorsage.bqr.png", "tumorsage.bqr.png"),
-                input(expectedRuntimeBucketName() + "/linx_germline/results/", "linx_germline"),
-                input(expectedRuntimeBucketName() + "/linx/results/", "linx"),
-                input(expectedRuntimeBucketName() + "/virusintrprtr/tumor.virus.annotated.tsv", "tumor.virus.annotated.tsv"),
-                input(expectedRuntimeBucketName() + "/cuppa/tumor.cuppa.vis_data.tsv", "tumor.cuppa.vis_data.tsv"),
-                input(expectedRuntimeBucketName() + "/cuppa/tumor.cuppa.pred_summ.tsv", "tumor.cuppa.pred_summ.tsv"),
-                input(expectedRuntimeBucketName() + "/cuppa/tumor.cuppa.vis.png", "tumor.cuppa.vis.png"),
-                input(expectedRuntimeBucketName() + "/lilac/tumor.lilac.tsv", "tumor.lilac.tsv"),
-                input(expectedRuntimeBucketName() + "/lilac/tumor.lilac.qc.tsv", "tumor.lilac.qc.tsv"),
-                input(expectedRuntimeBucketName() + "/peach/reference.peach.haplotypes.best.tsv", "reference.peach.haplotypes.best.tsv"),
-                input(expectedRuntimeBucketName() + "/sigs/tumor.sig.allocation.tsv", "tumor.sig.allocation.tsv"));
+                input(TestInputs.SOMATIC_BUCKET + "/sage_germline/results/tumorsage.gene.coverage.tsv", "tumorsage.gene.coverage.tsv"),
+                input(TestInputs.SOMATIC_BUCKET + "/sage_somatic/results/referencesage.bqr.png", "referencesage.bqr.png"),
+                input(TestInputs.SOMATIC_BUCKET + "/sage_somatic/results/tumorsage.bqr.png", "tumorsage.bqr.png"),
```

Comment:
Do we care that these names aren't quite what they would be in the actual pipeline run output? Probably not, right?

Reply:
No I don't think we care for these tests. Any change missed due to it would be covered by smoke and/or end-to-end tests later, so no real issue I think.